app/src/main/java/ch/epfl/sdp/appart/favorites/FavoriteViewModel.java

```diff
@@ -66,4 +56,50 @@ public void initHome() {
     public MutableLiveData<List<Card>> getFavorites() {
         return lFavorites;
     }
+
+    /**
+     * Fetches info from the DatabaseService and if successful updates the local database
+     */
+    private CompletableFuture<Void> fetch() {

```

Comment:
I don't really get the point of extracting everything in a new private method. Right now initHome() is doing nothing else than calling fetch(). As such, shouldn't the content of fetch simply be in initHome() ?

Reply:
Leftover from my other PR, where we have to first load locally and then fetch from server, in that case it's better to keep this function separate. I decided to leave it like this so that I don't have to rewrite when I'm going to work on the next things